### DISCUSSION TITLE
Provide support for the LaRS dataset in MMDet and evaluate it using the Mask2Former algorithm. 给 MMDet 支持 LaRS 数据集，并采用 Mask2Former 进行评估

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#680] [MMDetection]
# Provide support for the LaRS dataset in MMDet and evaluate it using the Mask2Former algorithm. 
 给 MMDet 支持 LaRS 数据集，并采用 Mask2Former 进行评估 

Provide support for the [LaRS dataset](https://arxiv.org/pdf/2308.09618.pdf) in MMDet and evaluate it using the Mask2Former algorithm.

给 MMDet 支持 [LaRS 数据集](https://arxiv.org/pdf/2308.09618.pdf)，并采用 Mask2Former 进行评估

## Task Information

**Repository**：MMDetection

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：24 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：2023年09月01日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by yaoshanliang** [comment]
Any response?
